### PR TITLE
Allow to select a default value for a RadioGroup

### PR DESCRIPTION
**Pull request by eloimrg**
This was made with Mahdiye

Do add a default value to a RadioGroup element, simply add `defaultValue` as a props. The value should match one of the value in the options array

### CODE REVIEW

**Comment by mnicole** [comment]
snapshot test is failing now, could you update please?

**Comment by eloimrg** [comment]
It's updated

File: _stories/molecules/RadioGroup/index.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const component = () => (
         context={optionalSelect('Context', contextOptions, '')}
         name="my-radio-group"
         options={[{ label: 'Foo', value: 'foo' }, { label: 'Bar', value: 'bar' }]}
+        defaultValue="bar"
```
Comment (mnicole) [inline _stories/molecules/RadioGroup/index.js]: can you make this changeable in storybook? `defaultValue={text('defaultValue', 'bar'}`
and you need to `import { text } from '@storybook/addon-knobs';`

Comment (eloimrg) [reply _stories/molecules/RadioGroup/index.js]: Indeed, I tried at first and it wasn't working

File: components/molecules/RadioGroup.jsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ const RadioGroup = ({ context, size, options, onChange, onBlur, name, className,
                     context={context}
                     onChange={onChange}
                     value={value}
+                    defaultChecked={defaultValue == value}
```
[the remark this answers is not in the document]
Comment (eloimrg) [reply components/molecules/RadioGroup.jsx]: thank you it's fixed :)

**Pull request merged**